### DISCUSSION TITLE
Onlyoffice

### DISCUSSION THREAD

**Discussion by joaopinto14** [nextcloud/all-in-one#3016] [Ideas] [answered]
Nextcloud Office doesn't work on my installation and I'm more used to onlyoffice.

I know they removed onlyoffice, I don't know if it was completely gone or not but I would like it to be available again or even a detailed tutorial to add onlyoffice to the nextcloud all-in-one instance.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/2461
